app/harvesters/ia_harvester.rb

```diff
@@ -0,0 +1,178 @@
+require 'json'
+
+##
+# A harvester implementation for Internet Archive
+#
+class IaHarvester
+  include Krikri::Harvester
+
+  DEFAULT_THREAD_COUNT = 10
+  DEFAULT_MAX_RECORDS = 0
+
+  DOWNLOAD_BASE_URI = 'http://archive.org/download'
+
+  ##
+  # Initialize, and set default options as appropriate.
+  #
+  # @param opts [Hash] a hash of options as defined by {.expected_opts}
+  #
+  # @example
+  #    Typical instantiation, good for most cases:
+  #
+  #      IaHarvester.new(uri: 'http://archive.org/...')
```

Comment:
Should the documentation specify which URI parameters should not be included (e.g. `rows`, since this is set by `:threads`)?